src/ui/components/NetworkMonitor/RequestRow.tsx

```diff
@@ -51,7 +51,7 @@ export const RequestRow = ({
           [styles.future]: !isInPast,
           [styles.unloaded]: !isInLoadedRegion,
         })}
-        onClick={() => onClick(row.original)}
+        onClick={event => onClick(row.original, event)}
```

Comment:
This change still seems unnecessary. Nothing is using the event param, is it?

Reply:
Sorry, missed that one. Removing